src/App.tsx

```diff
@@ -8,7 +7,57 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [actualTodos, setActualTodos] = useState<Todo[]>([]);
```

Comment:
do not duplicate states - calculate actualTodos (use `useMemo`) and store them in variable

Reply:
I dont find an answer how to put data from server into variable :c